docs/adding-a-library.md

```diff
@@ -0,0 +1,322 @@
+---
+title: Adding your Library
+---
+
+# Integrating your library to the Hub
+
+## Introduction
+
+The Hugging Face Hub aims to facilitate the sharing of machine learning models, checkpoints, and artifacts.  This endeavor starts with the integration of its tool stack within downstream libraries, and we're happy to announce  the fruitful collaboration between Hugging Face and spaCy, AllenNLP, and Timm, among many other incredible libraries.
+
+**For a full list of supported libraries, see [the following table](/docs/libraries).**
+
+We believe the Hub is a step in the correct direction for several reasons. It offers:
+
+- Free model hosting for libraries and their users. Custom plans exist for users wishing to keep their models  private but keep the ease of use.
```

Comment:
Actually, private models are generally available now, without a need for a paid plan :)

Reply:
Heh that's a good point :) 

I'm actually wondering if I should remove the sentence completely. After all, we're still figuring out private models for 3rd party libraries (see https://github.com/huggingface/huggingface_hub/issues/85)